tests/test_expectation_maximization.py

```diff
@@ -0,0 +1,173 @@
+"""Test bayesian expectation maximization process."""
+
+import numpy as np
+import pytest
+
+from reconstructSPI.iterative_refinement import expectation_maximization as em
+
+
+@pytest.fixture
+def n():
+    """Get test array size value for consistency."""
+    return 2
+
+
+@pytest.fixture
+def test_ir(n):
+    """Instantiate IterativeRefinement class for testing."""
+    ex_ctf = {
+        "s": np.ones((n, n)),
+        "a": np.ones((n, n)),
+        "def1": 1.0,
+        "def2": 1.0,
+        "angast": 0.1,
+        "kv": 0.1,
+        "cs": 0.1,
+        "bf": 0.1,
+        "lp": 0.1,
```

Comment:
this is all unrealistic, but we will fix once there is the right function called. This test is just for shape.

Reply:
This placeholder can be replaced when the CTF backend is decided upon. Comments put in https://github.com/compSPI/reconstructSPI/issues/11